core/src/banking_stage/consumer.rs

```diff
@@ -684,6 +709,39 @@ impl Consumer {
         }
     }
 
+    pub fn check_fee_payer_unlocked(
```

Comment:
put this into a separate fn because I actually will be doing same operation in the new scheduler (#34699)